src/Project/Project.router.ts

```diff
@@ -290,6 +299,62 @@ export class ProjectRouter extends Router {
     return true
   }
 
+  async getContents(req: Request, res: Response) {
+    const projectId = server.extractFromReq(req, 'id')
+    const content = server.extractFromReq(req, 'content')
+
+    if (content === INDEX_HASH) {
+      return res.send(new TextDecoder().decode(scenePreviewMain))
```

Comment:
we could avoid using a `TextDecoder` here by passing `utf-8` as the second argument of `readFileSync`, ie:

```
const stringValue = fs.readFileSync('file.js', 'utf-8') 
```